cpp/benchmarks/io/parquet/parquet_reader_input.cpp

```diff
@@ -149,8 +148,9 @@ void BM_parquet_read_chunks(
                try_drop_l3_cache();
 
                timer.start();
+               auto reader = cudf::io::chunked_parquet_reader(byte_limit, read_opts);
```

Comment:
```suggestion
               auto reader = cudf::io::chunked_parquet_reader(byte_limit, read_opts);
               timer.start();
```
Do we want to move this line before `timer.start()` to exclude the reader construction time?

Reply:
I've thought about this and decided to include it. The reader object is not reusable, so IMO the time it takes to create it should be taken into account as part of the time needed to read a file. There's no way for the user to offset this cost.
In addition, the constructor might be doing a lot of preprocessing (e.g. to determine the chunks).
CC @nvdbaranec

Reply:
>  There's no way for the user to offset this cost.

good point